pkg/utils/erofs/erofs_linux.go

```diff
@@ -15,10 +15,12 @@ import (
 func Mount(bootstrapPath, fsID, mountPoint string) error {
 	mount := unix.Mount
 
-	opts := "fsid=" + fsID
+	opts := fmt.Sprintf("domain_id=%s,fsid=%s", "global-domain", fsID)
 	logrus.Infof("Mount erofs to %s with options %s", mountPoint, opts)
 	if err := mount("erofs", mountPoint, "erofs", 0, opts); err != nil {
-		return errors.Wrapf(err, "failed to mount erofs")
+		if err := mount("erofs", mountPoint, "erofs", 0, "fsid="+fsID); err != nil {
+			return errors.Wrapf(err, "failed to mount erofs")
```

Comment:
Why do you retry another mount if the L20 mount operation fails? It's better to add a comment to tell necessity and intention.

Reply:
If a mount with the option "domain_id" fails, this indicates that the kernel may not support EROFS shared domain, and will fall back to normal mount. I'll add a comment about that. Thanks.

Reply:
I see. Can it check return error code like unsupported or invalid ?